repositories/templates/jenkins-x-apps.yaml

```diff
@@ -334,3 +334,76 @@ spec:
   providerName: github
   org: jenkins-x-apps
   repo: jx-app-fossa
+---
+apiVersion: jenkins.io/v1
+kind: SourceRepository
+metadata:
+  annotations:
+    jenkins.io/last-build-number-for-master: "2"
```

Comment:
Remove the annotations, creationTimestamp, and scheduler fields from the new repos so that they match the other ones?